docs/api/browser-window.md

```diff
@@ -1810,6 +1810,13 @@ with `addBrowserView` or `setBrowserView`.
 **Note:** The BrowserView API is currently experimental and may change or be
 removed in future Electron releases.
 
+#### `win.setModalDialogOptions(options)`
```

Comment:
I'd like to bikeshed this name a bit. `modal` is a concept that has quite a lot of meanings already, in particular, `BrowserWindow` can be constructed with a `modal: true` parameter, which this method is completely unrelated to.

How about `win.setWebAuthnDialogOptions`? Do we expect to implement other overlay dialogs like this one in the future?

Reply:
This sets an attribute on a generic `ModalDialogHost` owned by `NativeWindow`. WebAuthn is currently the only usage of that, but it's a generic component and could certainly be used by other features. I don't feel strongly about any of this and am happy to name it whatever y'all feels is best.